TRANSLATING.md

```diff
@@ -0,0 +1,142 @@
+# Translate Script Lab to other languages
+
+This document describes how to create a new translation for Script Lab or improve an existing translation.
+
+## Topics
+* [Prerequisites](TRANSLATING.md#prerequisites)
+* [Understanding the Script Lab structure for translating](TRANSLATING.md#structure)
+* [Create a new translation](TRANSLATING.md/#create)
+* [Improve an existing translation](TRANSLATING.md#improve)
+* [Testing your translation](TRANSLATING.md#testing)
+* [Script Lab languages](TRANSLATING.md/#translations)
+
+<a id="prerequisites"></a>
+## Prerequisites
+
+Fork this project into your GitHub account and use branches to update existing files or add new files. When you are ready, send us a pull request. Please note, that if you are not employed by Microsoft and you have not already signed the Microsoft Contribution License Agreement, you will be asked to sign the agreement before your pull request is accepted.
```

Comment:
for "use branches", maybe be more specific?  I.e., "and create a branch for your translation work, which you will later use for a pull request"